resources/postcards/responses/postcard.yml

```diff
@@ -27,10 +27,10 @@ application/json:
           object: address
         url: "https://lob-assets.com/postcards/psc_208e45e48d271294.pdf?version=v1&expires=1619218302&signature=NfHHLBSr5tOHA_Z4kij4dKqZG8f3vMDtwvuFVeeF9pV_lylcjLsVVODhNCE5hR6-2slUr6t9WMNsi429Pj7_DA"
         carrier: USPS
-        front_template_id: null
```

Comment:
The [response in the docs](https://docs.lob.com/#postcards_create) explicitly defines these values as `null`, so why are we changing it here?

Reply:
iirc, I changed front_template_id to add a description, but then I ran into problems with the linter because it wanted a string but apparently `nullable` wasn't working...or sth

Reply:
or maybe i was in a fever dream cuz i just turned it back into null and it works fine -shrug- committing